weave-js/src/assets/icons/icon-not-visible.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+        d="M16.0094 7.50093C16.285 7.64426 16.3923 7.98388 16.2489 8.2595L15.7499 7.99999C16.2489 8.2595 16.2489 8.2595 16.2489 8.2595L16.2483 8.26066L16.2475 8.26216L16.2454 8.26619L16.2389 8.27827C16.2336 8.28808 16.2263 8.30139 16.217 8.31794C16.1984 8.35105 16.1717 8.39721 16.1366 8.45455C16.0665 8.56918 15.9629 8.72888 15.8238 8.91857C15.6492 9.15668 15.4177 9.44342 15.1252 9.74817C14.9693 9.9107 14.9649 10.1694 15.1242 10.3287L15.9227 11.1272C16.1424 11.3469 16.1424 11.7031 15.9227 11.9227C15.7031 12.1424 15.3469 12.1424 15.1272 11.9227L14.2028 10.9983C14.0642 10.8597 13.8462 10.8428 13.6825 10.9506C13.2538 11.2329 12.7672 11.4958 12.2202 11.7139C12.0051 11.7997 11.894 12.0434 11.9831 12.2572L12.3692 13.1836C12.4886 13.4704 12.353 13.7997 12.0663 13.9192C11.7795 14.0387 11.4502 13.9031 11.3307 13.6163L10.8457 12.4524C10.7728 12.2774 10.5871 12.1786 10.3997 12.2075C9.96043 12.2751 9.49406 12.3125 8.99987 12.3125C8.50573 12.3125 8.0394 12.2751 7.60016 12.2075C7.41274 12.1786 7.22707 12.2774 7.15414 12.4525L6.66919 13.6163C6.54971 13.9031 6.22038 14.0387 5.93361 13.9192C5.64685 13.7997 5.51124 13.4704 5.63073 13.1836L6.01673 12.2572C6.10581 12.0434 5.99481 11.7997 5.77967 11.714C5.2326 11.4959 4.74603 11.2329 4.31734 10.9507C4.15364 10.8429 3.93563 10.8598 3.79704 10.9983L2.87264 11.9227C2.65297 12.1424 2.29682 12.1424 2.07715 11.9227C1.85748 11.7031 1.85748 11.3469 2.07715 11.1272L2.8756 10.3288C3.03488 10.1695 3.03052 9.91076 2.87456 9.74823C2.58212 9.44346 2.35058 9.15669 2.17596 8.91857C2.03685 8.72888 1.93322 8.56918 1.86313 8.45455C1.82807 8.39721 1.80134 8.35105 1.78272 8.31794C1.7734 8.30139 1.7661 8.28808 1.7608 8.27827L1.75433 8.26619L1.7522 8.26216L1.75141 8.26066C1.75141 8.26066 1.75081 8.2595 2.24983 8.00001L1.75081 8.2595C1.60749 7.98388 1.71474 7.64426 1.99036 7.50093C2.26556 7.35783 2.60457 7.46453 2.74827 7.73922L2.75043 7.74323C2.75279 7.74761 2.75703 7.75536 2.76317 7.76627C2.77545 7.7881 2.79531 7.82251 2.82294 7.8677C2.87824 7.95815 2.96446 8.09142 3.08316 8.25329C3.32098 8.57759 3.68661 9.01314 4.19215 9.44895C5.19856 10.3166 6.76008 11.1875 8.99987 11.1875C11.2397 11.1875 12.8012 10.3166 13.8076 9.44895C14.3131 9.01314 14.6788 8.57759 14.9166 8.25329C15.0353 8.09142 15.1215 7.95815 15.1768 7.8677C15.2044 7.82251 15.2243 7.7881 15.2366 7.76627C15.2427 7.75536 15.247 7.74761 15.2493 7.74323L15.2508 7.74048L15.2512 7.73967C15.3949 7.46497 15.7342 7.35783 16.0094 7.50093Z"
+        fill="#2B3038" />
```

Comment:
we want fill to be "currentColor"

you can use jamies addIcon script located here
https://github.com/wandb/devtools/blob/main/docs/addicon.md

in his devtools repo

Reply:
oh sick, didn't know about this

Reply:
i think we still want these fills to be "currentColor"

Reply:
oh I modified the wrong fill

Reply:
nice looks good now